frontend/src/component/feature/FeatureView/FeatureVariants/FeatureEnvironmentVariants/EnvironmentVariantsModal/VariantForm/VariantForm.tsx

```diff
@@ -86,8 +86,12 @@ const StyledFieldColumn = styled('div')(({ theme }) => ({
     },
 }));
 
-const StyledInput = styled(Input)(() => ({
+const StyledInput = styled(Input)(({theme}) => ({
```

Comment:
Tiny styling / formatting nitpick.

```suggestion
const StyledInput = styled(Input)(({ theme }) => ({
```